api/v1alpha1/foundationloadbalancerconfig_types.go

```diff
@@ -145,6 +145,10 @@ type FoundationLoadBalancerNodeStatus struct {
 
 // FoundationLoadBalancerConfigStatus describes the observed state of the Foundation Load Balancer.
 type FoundationLoadBalancerConfigStatus struct {
+	// Version describes the current version of the Foundation Load Balancer.
```

Comment:
should we say 'most-recent observed version', so as to avoid the impression that the value is definitive? netop<-->vpxd reflection may be down, and this value could be outdated because of that.

Reply:
+1

Reply:
Status is always implicitly observed state. It does not need to explicitly state most currently observed version as "observed" should be understood.